test/image/mocks/splom_large.json

```diff
@@ -297,6 +297,14 @@
     "yaxis": {
       "zerolinecolor": "red",
       "zerolinewidth": 2
+    },
+    "xaxis2": {
+      "zeroline": false
+    },
+    "yaxis2": {
```

Comment:
which currently renders as:

![image](https://user-images.githubusercontent.com/6675409/45568628-23e33f00-b82b-11e8-856e-2f90043cc296.png)

notice the missing zero line on the second row from the top.